### PR TITLE
FlowCleanup: Remove flows with 0 traffic added more than a year back

### PR DESCRIPTION
**Pull request by ddc22**
Related to #

### Removed flows
- onboarding-2023-pricing-grid 
- rewind-switch
- plan-no-domain 
 - ecommerce-2y 
- ecommerce-3y 
- link-in-bio-domain
- podcasts 
- connect-domain 
- google-transfer 
- hosted-site-migration

## Why are these changes being made?

* Cleanup to remove tech debt: pau2Xa-605-p2

## Testing Instructions

* Unit tests should not be broken
* Removal of flows so nothing much to test. Maybe checkout some flows in both frameworks.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by aneeshd16 — APPROVED** [review]
Based on the fresh audit data (pau2Xa-605-p2#comment-15697), I think this PR is good to go in it's current state. LGTM!

**Comment by tyxla** [comment]
Great cleanup 👏

File: client/signup/steps/rewind-migrate/index.jsx [hunk 1]
```diff
@@ -1,99 +0,0 @@
-import { Card } from '@automattic/components';
-import { localize } from 'i18n-calypso';
-import { get } from 'lodash';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import ActivityLogRewindToggle from 'calypso/my-sites/activity/activity-log/activity-log-rewind-toggle';
```
Comment (tyxla) [inline client/signup/steps/rewind-migrate/index.jsx:7]: This could be an additional opportunity for removal.

Comment (tyxla) [reply client/signup/steps/rewind-migrate/index.jsx:7]: Done in #100266, @ddc22 do you mind taking a look?

Comment (ddc22) [reply client/signup/steps/rewind-migrate/index.jsx:7]: Nice catch @tyxla sorry I missed this.

Comment (tyxla) [reply client/signup/steps/rewind-migrate/index.jsx:7]: No problem, it's long merged already 😉

**Pull request merged**